modules/base/rendering/renderablemodel.cpp

```diff
@@ -366,14 +363,14 @@ RenderableModel::RenderableModel(const ghoul::Dictionary& dictionary)
             }
 
             if (wasFound) {
-                _animationTimeScale = static_cast<double>(
-                    convertTime(1.0, timeUnit, TimeUnit::Second)
-                );
+                _animationTimeScale = convertTime(1.0, timeUnit, TimeUnit::Second);
             }
             else {
-                std::string message = std::format("The given unit name '{}' does not "
-                    "match any currently supported unit names", stringUnit);
-                LERROR(message);
+                LERROR(std::format(
+                    "The given unit name '{}' does not match any currently supported "
+                    "unit names",
+                    stringUnit
+                ));
```

Comment:
```suggestion
                LERROR(std::format(
                    "The given unit name '{}' does not match any currently supported "
                    "unit names", stringUnit
                ));
```
more relaxed rules for logs